config.json

```diff
@@ -824,6 +824,17 @@
       ]
     },
     {
+      "uuid": "7f4d5743-7ab8-42ca-b393-767f7e9a4e97",
+      "slug": "complex-numbers",
+      "core": false,
+      "unlocked_by": "leap",
+      "difficulty": 6,
+      "topics": [
+        "Tuples",
```

Comment:
Good choice.

We are trying to standardize topics across all tracks and decided to use `snake_case` notation as a naming convention. List of of common topics can be found here - https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt 
Could you please convert them to 'snake_case'